### ISSUE TITLE
[Stats Migration] Alt text for images does not migrate to Torus.

### ISSUE THREAD

**Issue by meturner1122** [Simon-Initiative/oli-torus#3669]
**Describe the bug**
Alt text for images does not migrate to Torus.

**Screenshots**
Page: Exploratory Data Analysis (2 of 2)
<img width="779" alt="Screen Shot 2023-06-09 at 10 46 57 AM" src="https://github.com/Simon-Initiative/oli-torus/assets/135067656/74abe81d-bb84-4564-8b46-bb7e9311c1a3">
<img width="858" alt="Screen Shot 2023-06-09 at 10 46 48 AM" src="https://github.com/Simon-Initiative/oli-torus/assets/135067656/9b3e05da-04ef-4a41-9826-d8460313dfb8">

[Torus](https://tokamak.oli.cmu.edu/sections/gsu_stats_main_campus_spring_2/page/589t9_exploratory_data_analysis_2_of_xhtvx)
[Legacy](https://oli.cmu.edu/jcourse/workbook/activity/page?context=9b26d5ca0a0001dc6bfba1554e49565f)
Echo
SVN

**Environment:** Version 23

**Other Occurrences:**
Extra Problems (Unit 2)
Histogram (3 of 3)
Histogram: Shape
Measures of Center (1 of 2)
Measures of Center (2 of 2)
Measures of Spread (1 of 5)
Measures of Spread (2 of 5)
Pictograms
Stemplot

**Comment by darrensiegel** [comment]
This is not an issue. Legacy OLI rendered alt text as a tooltip in addition to setting it as the alt text.  Torus does not render the tooltip.  Our system should not render alt text as a student visible tooltip as that would  influence authors in how the word the alt text.  Alt text should be worded spécifically for screen readers and missing images.
